File: src/main/java/net/ndrei/teslapoweredthingies/gui/IDualTankMachine.java
package net.ndrei.teslapoweredthingies.gui;

import net.minecraftforge.fluids.Fluid;

/**
 * Created by CF on 2017-01-11.
 */
public interface IDualTankMachine {
    Fluid getLeftTankFluid();
    float getLeftTankPercent();

    Fluid getRightTankFluid();
    float getRightTankPercent();
}

File: src/main/java/net/ndrei/teslapoweredthingies/gui/IDualTankMachine.kt
package net.ndrei.teslapoweredthingies.gui

import net.minecraftforge.fluids.Fluid

/**
 * Created by CF on 2017-06-30.
 */
interface IDualTankMachine {
    val leftTankFluid: Fluid
    val leftTankPercent: Float

    val rightTankFluid: Fluid
    val rightTankPercent: Float
}
